From e060a1ddee76f543bd4cdc463d24b772c757231d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <cynerd@email.cz>
Date: Fri, 15 Feb 2019 14:50:17 +0100
Subject: [PATCH] i2c-tools: fix compilation for python3

There seems to be missing dependency for python3 because this package
failed compilation beceause of missing python3.
---
 utils/i2c-tools/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/utils/i2c-tools/Makefile b/utils/i2c-tools/Makefile
index 88dd6b3f6..3e4397fff 100644
--- a/utils/i2c-tools/Makefile
+++ b/utils/i2c-tools/Makefile
@@ -18,7 +18,7 @@ PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_HASH:=db5e69f2e2a6e3aa2ecdfe6a5f490b149c504468770f58921c8c5b8a7860a441
 
 PKG_BUILD_PARALLEL:=1
-PKG_BUILD_DEPENDS:=PACKAGE_python-smbus:python
+PKG_BUILD_DEPENDS:=PACKAGE_python-smbus:python PACKAGE_python3-smbus:python3
 
 PKG_MAINTAINER:=Daniel Golle <daniel@makrotopia.org>
 PKG_LICENSE:=GPLv2
@@ -121,5 +121,5 @@ endef
 $(eval $(call BuildPackage,i2c-tools))
 $(eval $(call PyPackage,python-smbus))
 $(eval $(call BuildPackage,python-smbus))
-$(eval $(call PyPackage,python3-smbus))
+$(eval $(call Py3Package,python3-smbus))
 $(eval $(call BuildPackage,python3-smbus))
-- 
2.20.1

